test/project/pkg/webhook/default_server/firstmates/mutating/create_update_webhook.go

```diff
@@ -22,6 +22,8 @@ import (
 	crewv1 "sigs.k8s.io/kubebuilder/test/project/pkg/apis/crew/v1"
 )
 
+// +kubebuilder:webhook:groups=apps,resources=deployments,verbs=CREATE;UPDATE
+// +kubebuilder:webhook:name=test-name,path=/test-path,type=mutating,failure-policy=Fail
```

Comment:
If I define multiple webhooks in the same file, how will that look ? How do we know that these two lines define the same webhook ?

Reply:
This is for one webhook no matter how many lines.
```
// +kubebuilder:webhook: ...
// +kubebuilder:webhook: ...
```

Reply:
This is for multiple webhooks when there are newlines in between
```
// This is for webhook foo
// +kubebuilder:webhook: ...
// +kubebuilder:webhook: ...
code for foo

// This is for webhook bar
// +kubebuilder:webhook: ...
// +kubebuilder:webhook: ...
code for bar
```